src/algebraic_topology/dold_kan/compatibility.lean

```diff
@@ -176,6 +176,119 @@ end
 
 lemma equivalence_functor : (equivalence hF hG).functor = F ⋙ eB.inverse := rfl
 
+omit hG hF
+
+/-- The isomorphism `eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor` deduced
+from the counit isomorphism of `e'`. -/
+@[simps hom_app]
+def τ₀ : eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor :=
+calc eB.functor ⋙ e'.inverse ⋙ e'.functor
+  ≅ eB.functor ⋙ 𝟭 _ : iso_whisker_left _ e'.counit_iso
+... ≅ eB.functor : functor.right_unitor _
```

Comment:
```suggestion
        ≅ eB.functor ⋙ 𝟭 _ : iso_whisker_left _ e'.counit_iso
...     ≅ eB.functor : functor.right_unitor _
```
What do you think? I'm not sure there are hard and fast rules for spacing in calc blocks and I think that my suggestion is a bit more readable.